recipes/colorlog/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install --single-version-externally-managed --record record.txt
+if errorlevel 1 exit 1
```

Comment:
same story here: you don't need bld.bat anymore (as of conda-build 1.20.0). the build: script: in meta.yaml suffices.